### ISSUE TITLE
CMD+B for linking into Gradle Precompiled Script Plugins source takes too many steps

### ISSUE THREAD

**Issue by ZakTaccardi** [gradle/gradle#18044]
Given I have some pre-compiled script plugin under:

```
buildSrc/src/main/kotlin/com/conventions/library/androidAllowSamePackageName.gradle.kts
```

And I can reference it like this:

```kts
plugins {
    com.conventions.library.androidAllowSamePackageName
}
```

If I `cmd` + `B` on `androidAllowSamePackageName` from the IDE, I am taken to:

```kts
/**
 * The `com.conventions.library.androidAllowSamePackageName` plugin implemented by [com.conventions.library.AndroidAllowSamePackageNamePlugin].
 */
val `ComConventionsLibraryPluginGroup`.`androidAllowSamePackageName`: PluginDependencySpec
    get() = plugins.id("com.conventions.library.androidAllowSamePackageName")
``` 
I can then `cmd` + `b` on `[com.conventions.library.AndroidAllowSamePackageNamePlugin]`, where I'm taken to a file like:

```kts
/**
 * Precompiled [androidAllowSamePackageName.gradle.kts][com.conventions.library.AndroidAllowSamePackageName_gradle] script plugin.
 *
 * @see com.conventions.library.AndroidAllowSamePackageName_gradle
 */
class AndroidAllowSamePackageNamePlugin : org.gradle.api.Plugin<org.gradle.api.Project> {
```

It is not until I `cmd` + `b` on `[com.conventions.library.AndroidAllowSamePackageName_gradle]` that I can get to the source file - `buildSrc/src/main/kotlin/com/conventions/library/androidAllowSamePackageName.gradle.kts`.

This is too many steps to get to the source `gradle.kts` file. 

**Desired behavior**
I would like this to be reduced to 1-2 steps, where either:

```kts
plugins {
  // cmd + b here does one of the following
  // 1. goes directly to the source file (disadvantage here is that you cannot get to the generated code at all)
  // 2. should and IDE dropdown to select either the generated code or the source file
  // 3. goes directly to the current generated code, but the docs for it contain the `[com.conventions.library.AndroidAllowSamePackageName_gradle]` which allows deep linking into the source, making it a two step process
  com.conventions.library.androidAllowSamePackageName
}
```

I personally like number 3 the best, but I'd be happy with any of the 3.

**Comment by vlsi** [comment]
Here's a relevant IDEA issue: https://youtrack.jetbrains.com/issue/IDEA-269370

**Comment by h0tk3y** [comment]
Sorry for the late reply.

We believe this issue is no longer relevant. Please file a new issue if necessary.

***

The [relevant issue in IntelliJ IDEA](https://youtrack.jetbrains.com/issue/IDEA-269370) has been fixed.
